GAE/src/org/akvo/flow/xml/XmlQuestionGroup.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Copyright (C) 2019 Stichting Akvo (Akvo Foundation)
+ *
+ *  This file is part of Akvo FLOW.
+ *
+ *  Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ *  the GNU Affero General Public License (AGPL) as published by the Free Software Foundation,
+ *  either version 3 of the License or any later version.
+ *
+ *  Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Affero General Public License included below for more details.
+ *
+ *  The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ */
+
+package org.akvo.flow.xml;
+
+import java.util.Arrays;
+import java.util.TreeMap;
+
+import org.waterforpeople.mapping.app.gwt.client.survey.QuestionDto;
+import org.waterforpeople.mapping.app.gwt.client.survey.QuestionGroupDto;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+/* Class for working with XML like this:
+<questionGroup repeatable = "false">
+    <heading>G2</heading>
+    <question>
+    </question>
+</questionGroup>
+ */
+public class XmlQuestionGroup {
+
+    @JacksonXmlElementWrapper(useWrapping = false)
+    private XmlQuestion[] question;
```

Comment:
```suggestion
    private XmlQuestion[] questionList
```

Reply:
No, this name is the name used in the xml.

Reply:
Ahh. ok thats the reason I hadnt thought of it! then I guess leave it as is.